packages/core/useIntersectionObserver/index.ts

```diff
@@ -70,33 +70,33 @@ export function useIntersectionObserver(
 
   const stopWatch = isSupported.value
     ? watch(
-      () => [targets.value, unrefElement(root as MaybeComputedElementRef), isActive.value] as const,
-      ([targets, root]) => {
-        cleanup()
-        if (!isActive.value)
-          return
-
-        if (!targets.length)
-          return
-
-        const observer = new IntersectionObserver(
-          callback,
-          {
-            root: unrefElement(root),
-            rootMargin,
-            threshold,
-          },
-        )
-
-        targets.forEach(el => el && observer.observe(el))
-
-        cleanup = () => {
-          observer.disconnect()
-          cleanup = noop
-        }
-      },
-      { immediate, flush: 'post' },
-    )
+        () => [targets.value, unrefElement(root as MaybeComputedElementRef), isActive.value] as const,
```

Comment:
Seems like there is a few lint issues that need fixing in the main repo, though unsure why it only started to popup now